bundles/org.openhab.core.ephemeris/src/main/java/org/openhab/core/ephemeris/internal/EphemerisManagerImpl.java

```diff
@@ -107,7 +111,6 @@ public class EphemerisManagerImpl implements EphemerisManager, ConfigOptionProvi
     public EphemerisManagerImpl(final @Reference LocaleProvider localeProvider) {
         this.localeProvider = localeProvider;
 
-        final Bundle bundle = FrameworkUtil.getBundle(getClass());
```

Comment:
I would suggest to add `BundleContext bc` to the constructor parameters and use `Bundle bundle = bundleContext.getBundle();` here. I don't know why it was done in this strange way.

Reply:
Done.